Kubernetes/HOWTO-on-prem.md

```diff
@@ -0,0 +1,103 @@
+# Kubernetes On-Premises Deployment From Scratch -- Start to Finish #
+This guide will walk you through deploying *Kubernetes 1.8* on a Linux master and join two Windows nodes to it without a cloud provider.
+
+**Note**: The guide will assume a local working directory of `~/kube` for the Kubernetes setup. If you choose a different directory, just replace any references to that path.
+
+## Building the Binaries ##
+We will begin with a Linux master node and a Windows worker node on RS3+, both of which are assumed to be VMs.
+
+First, install all of the pre-requisites on the master:
+
+    # apt-get install git build-essentials docker.io

```

Comment:
This requires root, so perhaps we should add 'sudo' to the front of this command for clarity. Alternatively, if there is an assumption that we will be elevated to root during all of this guide, then we could just have a pre-requisite step to execute 'sudo -s' or similar to get them to a root shell.

Reply:
The `#` was a hint at that (vs. the standard `$`), but I will clarify since it's probably too subtle, especially for those unfamiliar with Linux!